src/forecast.jl

```diff
@@ -0,0 +1,31 @@
+export Forecast, 
+    Scenarios
+
+struct Forecast
+
+end
+
```

Comment:
Pensei em de repente colocar o "name" dentro da struct de Forecast e retirar da de cenários.
Olhando o formato de entrada/saída, a struct de Forecast irá sempre ter a struct de cenários na entrada, e terá na de saída apenas se for uma previsão probabilística. Acho que talvez fique mais claro colocar então "name" na struct mais externa (forecast no caso). O que acha?